bescort.lic

```diff
@@ -172,6 +178,7 @@ class Bescort
   def search
     fput('search')
     pause
```

Comment:
Not sure how that pause got in there.  I didn't intend to edit that part.  Must have been an errant keystroke which duplicated the line